mrp_product_variants/models/mrp_bom.py

```diff
@@ -86,6 +86,26 @@ def _bom_explode(self, bom, product, factor, properties=None, level=0,
             master_bom=master_bom, production=production)
         return result, result2
 
+    def _check_product_suitable(self, check_attribs, component_attribs):
+        """ Check if component is suitable for given attributes
+        @param production_attribs: Production variant attribute id list
+        @param component_attribs: Component defined attributes to check
+        @return: Component validity
+        """
+        if not check_attribs:
```

Comment:
What is the sense of this? If the argument `check_attribs` is False, which can be evaluated outside this method, directly don't call the method